### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by flaviuc** [spjeff/spbestwarmup#33]
What's the purpose of loading each subsite and each site collection?
Most of the warm-up can be efficiently performed by loading only 1 site in a respective web app or web app zone, probably preferably the root site collection.
What's the added benefit of making a request to all SPSites (and optionally to all subsites as well)?

**Comment by spjeff** [comment]
Good point.  Helps build SPLIST cache for smaller systems.  Optional feature, less needed on larger farms.  Thank you.
